src/pages/Landing/AboutUs.jsx

```diff
@@ -0,0 +1,124 @@
+
+const AboutUs = () => {
+    return (
+        <div className={styles.bgContainer}>
+
+            <div className={styles.mdContainer}>
+                <div className={styles.titles}>
+                    <h3 className={styles.title}>Sobre nós</h3>
+                    <p className={styles.subtitle}>Sabe mais sobre o nosso projecto.</p>
+                </div>
+                <p className={styles.text}>
+                    Vejo em Casa é uma plataforma sem fins lucrativos de conteúdos diversificados em formato live streaming, de cariz solidário e desenvolvida por uma equipa de voluntários. Este projeto integra o movimento tech4COVID19, cujo propósito é a criação de soluções que atenuem as consequências desta pandemia.
+                    <br /><br />
+                    Sentimos que este é o momento ideal para dar a volta por cima, para conhecer talentos e conteúdos do melhor que há, para partilhar e, acima de tudo, para ajudar. Por isso, todos os nossos eventos oferecem a oportunidade ao público de apoiar diretamente não só os criadores, mas também o movimento Tech4COVID19 e as suas iniciativas.
+                </p>
+                <div className={styles.powered}>Powered by:</div>
+                <img src={logo} alt='Logotipo do #tech4COVID19' className={styles.logo} />
+            </div>
+
+            <div className={styles.smContainer}>
+                <div className={styles.icons}>
+                    <div>
+                        <img 
+                            src={icon06} 
+                            alt='icon que simboliza tecnologia' 
+                            className={styles.icon}
+                            style={{width: '78px'}}
+                        />
+                        <p style={{color: '#FD641E'}}>TECNOLOGIA</p>
+                    </div>
+                    <div>
+                        <img 
+                            src={icon07} 
+                            alt='icon que simboliza música' 
+                            className={styles.icon}
+                            style={{width: '78px'}}
+                        />
+                        <p style={{color: '#EB2907'}}>MÚSICA</p>
+                    </div>
+                    <div>
+                        <img 
+                            src={icon08} 
+                            alt='icon que simboliza arte e design' 
+                            className={styles.icon}
+                            style={{width: '67px'}}
+                        />
+                        <p style={{color: '#55BFD8'}}>ARTE E DESIGN</p>
+                    </div>
+                    <div>
+                        <img 
+                            src={icon09} 
+                            alt='icon que simboliza ciência' 
+                            className={styles.icon}
+                            style={{width: '57px'}}
+                        />
+                        <p style={{color: '#2C5AA6'}}>CIÊNCIA</p>
```

Comment:
The same here about CSS styles.

Reply:
I agree with you, and I believe we can create trello tasks to solve this later. For the mvp and since we want to have a landing page asap, I would approve this PR. Do you agree?

Reply:
Yup, @sexta13 I agree. Those comments shouldn't be an impediment do merge the PR.